Rdkafka.xs

```diff
@@ -392,6 +392,15 @@ krdm_offset(msg)
     OUTPUT:
         RETVAL
 
+long
+krdm_timestamp(msg)
+        rd_kafka_message_t* msg
+    CODE:
+	rd_kafka_timestamp_type_t tstype;
+        RETVAL = rd_kafka_message_timestamp(msg, &tstype);
```

Comment:
Should probably check `tstype` here and act appropriately?

Reply:
Maybe the perl-facing function could accept an additional optional argument, which must be a scalar reference, and if it exists, then it should be filled with the tstype value.